acinclude.m4

```diff
@@ -1385,7 +1385,7 @@ AC_DEFUN([CURL_CHECK_WIN32_LARGEFILE], [
       AC_COMPILE_IFELSE([
         AC_LANG_PROGRAM([[
         ]],[[
-          #if !defined(_WIN32_WCE) && (defined(__MINGW32__) || defined(_MSC_VER))
+          #if !defined(_WIN32_WCE) && defined(__MINGW32__)
```

Comment:
Wouldn't we want to default to the large file API instead of the small? Defaulting to large breaks in an obvious way for the user if there is no support for large and there is some build mistake.